in_toto/verifylib.py

```diff
@@ -222,7 +221,10 @@ def run_all_inspections(layout):
 
     # Dump the inspection link file for auditing
     # Keep in mind that this pollutes the verifier's (client's) filesystem.
-    filename = FILENAME_FORMAT_SHORT.format(step_name=inspection.name)
+    # ignore snakecase for pylint to satisfy line limit
+    # pylint: disable=C0103
+    m = in_toto.models.link
+    filename = m.FILENAME_FORMAT_SHORT.format(step_name=inspection.name)
```

Comment:
[ping](https://github.com/in-toto/in-toto/pull/379#pullrequestreview-459873487)

Reply:
oops, looks like I missed one